go.mod

```diff
@@ -22,7 +22,6 @@ require (
 	github.com/99designs/keyring v1.2.2
 	github.com/AlecAivazis/survey/v2 v2.3.7
 	github.com/DATA-DOG/go-sqlmock v1.5.0
-	github.com/Pallinder/go-randomdata v1.2.0
```

Comment:
I think we should not add external libraries for everything. Here it was used only for random names.
Which I find overkilled to have a library for that. We can as well just use the hostname.